src/Model.elm

```diff
@@ -98,6 +102,20 @@ update msg model =
         CreateFail err ->
             ( { model | error = True, errorMsg = (toString err) }, Cmd.none )
 
+        DeleteRecord recordId ->
+            ( model, deleteRecord recordId )
+
+        DeleteRecordSucceed { data } ->
+            ( { model
+                | records = removeRecordFromList data model.records
+                , error = False
+              }
+            , Cmd.none
```

Comment:
I'm wondering if we shouldn't trigger a full refresh of the list instead of just removing the entry from our local one?